client/extensions/woocommerce/app/dashboard/store-location-setup-view.js

```diff
@@ -148,6 +141,18 @@ class StoreLocationSetupView extends Component {
 			// No need to set isSaving to false here - we're navigating away from here
 			// and setting isSaving to false will just light the button up again right
 			// before the next step's dialog displays
+
+			// mc stat 32 char max :P
+			analytics.mc.bumpStat( 'calypso_woo_store_setup_country', this.state.address.country );
```

Comment:
Just a note that I logged an issue at https://github.com/Automattic/wp-calypso/issues/21336, so that later we can update this to use a method that ignores test sites.

Reply:
Please avoid using bare analytics lib calls. Testing becomes much more difficult, since analytics assumes window is available. We do have analytics middleware available, which allows us to dispatch a redux action, or add additional meta data so we can fire this off. See: https://github.com/Automattic/wp-calypso/tree/master/client/state/analytics and let us know if you need help walking through that